client/src/pages/Authentication/Authentication.jsx

```diff
@@ -27,7 +27,6 @@ const Authentication = () => {
       try {
         await doSignInWithEmailAndPassword(logInEmail, logInPassword);
         navigate("/home");
-        console.log("signed in");
```

Comment:
remove consoles while pushing

Reply:
had removed after self review. Ill review before making pr in future!